sdk/monitor/monitor-query/review/monitor-query.api.md

```diff
@@ -13,15 +13,12 @@ export type AggregationType = "None" | "Average" | "Count" | "Minimum" | "Maximu
 
 // @public
 export interface BatchQuery {
-    azureResourceIds?: string[];
+    additionalWorkspaces?: string[];
```

Comment:
`additionalWorkspaceIds` maybe? Unless this is how all languages called it

Reply:
Yes this is across all languages